lib/ansible/plugins/doc_fragments/action_common_attributes.py

```diff
@@ -11,84 +10,58 @@ class ModuleDocFragment(object):
     # Standard documentation fragment
     DOCUMENTATION = r'''
 attributes:
-    action:
-      description: Indicates this has a corresponding action plugin so some parts of the options can be executed on the controller
-      support: none
+    check_mode:
+      description: Can run in check_mode and return changed status prediction withought modifying target
+    diff_mode:
+      description: Will return details on what has changed (or possibly needs changing in check_mode), when in diff mode
+    platform:
+      description: Target OS/families that can be operated against
+'''
+
+    actiongroups = r'''
```

Comment:
Aren't these normally uppercase?
```suggestion
    ACTIONGROUPS = r'''
```

Reply:
was forcing it to error so it wont merge by accident, will make upper when ready